r/R/utilities.R

```diff
@@ -2,28 +2,64 @@
 # subsetIds subsets a seurat object with the cell ids
 #
 subsetIds <- function(scdata, cells_id) {
-  meta_data_subset <- scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
+  meta_data_subset <-
+    scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
   current_cells <- rownames(meta_data_subset)
   scdata <- subset(scdata, cells = current_cells)
   return(scdata)
 }
 
-sendCellsetToApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
-  httr_query <- paste0('$[?(@.key == "', cell_set_key, '")]')
-  children <- list(list("$insert" = list(index = "-", value = new_cell_set)))
+sendCellsetToApi <-
```

Comment:
`sendCellsetToApi` looks identical to `updateCellSetsThroughApi` -- why do you have the same code twice under a different function?

Reply:
but it's not the same right? the queries are different, and the body as well. sendCellset... is used to create new cellsets based on expression, whereas updateCellsets replaces existing cellsets (useful when reclustering

Reply:
yeah, you are right, the queries and bodies are different. Better to have them as separate functions